backon/src/sleep.rs

```diff
@@ -91,3 +100,17 @@ impl Sleeper for GlooTimersSleep {
         gloo_timers::future::sleep(dur)
     }
 }
+
+/// The default implementation of `Sleeper` utilizes `embassy-time::Timer`.
+#[cfg(all(not(feature = "std"), feature = "embassy-sleep"))]
+#[derive(Clone, Copy, Debug, Default)]
+pub struct EmbassySleep;
```

Comment:
Please call it `EmbassySleeper` to align the name.

Reply:
Ok